resources/translations/fi.edn

```diff
@@ -1,16 +1,16 @@
 {:t
  {:actions {:accept-invitation "Hyväksy kutsu"
-            :accept-invitation-success [:div [:p "Liittyminen hakemukseen onnistui."] [:p "Seuraavaksi pääset katsomaan hakemuksen tietoja."]]
-            :accept-invitation-already-member [:div. [:p "Olet jo jäsen."] [:p "Seuraavaksi pääset katsomaan hakemuksen tietoja."]]
+            :accept-invitation-success [:div [:p "Hakemukseen liittyminen onnistui."] [:p "Siirry hakemukseen."]]
+            :accept-invitation-already-member [:div. [:p "Olet jo jäsen."] [:p "Siirry hakemukseen."]]
             :accept-licenses "Hyväksy käyttöehdot"
             :add-licenses "Lisää käyttöehto"
             :add-member "Lisää jäsen"
             :bundling-error "Resursseja ei voi hakea samalla hakemuksella."
-            :bundling-warning "Resursseja ei tavallisesti voi hakea samalla hakemuksella, koska niillä on eri lomake tai työvuo."
```

Comment:
All right then, how about "Kielivaroja ei tavallisesti voi hakea samalla hakemuksella, jos niillä on eri lomake tai työvuo." / "Applications of the language resources can't normally be bundled into one application if they have different forms or workflows."?

Reply:
There is a concrete case here that after changing the resources, there is now a conflict between the form or workflow so I'd put "koska" instead of "jos".

See the English common version
> The resources can't normally be bundled into one application because they have different forms or workflows."

Therefore the LBR version could use language resources here (or just resources).